docs/fireblocks.md

```diff
@@ -0,0 +1,92 @@
+# Fireblocks integration
+
+## Description
+
+The Fireblocks integration allows you to connect your wallet with Fireblocks using the RAW transaction signing method.
```

Comment:
suggestion: I see that this article is aimed at users that wish to use Fireblocks with the Headless Wallet.

However, with some changes it may also serve people are studying all possibilities of the Headless Wallet but do not yet know what Fireblocks is.

I believe that a brief explanation of Fireblocks with some explanatory links may help this other audience with little change.

Reply:
This should have been better detailed in the Acceptance Criteria. This is not supposed to be a complete and external guide. This is an internal guide that will be used by Campana to create the official tutorial, where I think we should have this explanation about Fireblocks